<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1033</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>阿西拜教学管理系统</string>
  </property>
  <property name="windowIcon">
   <iconset resource="res.qrc">
    <normaloff>:/ico/le.png</normaloff>:/ico/le.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>2</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>2</number>
    </property>
    <item>
     <widget class="QWidget" name="widget" native="true">
      <property name="minimumSize">
       <size>
        <width>100</width>
        <height>0</height>
       </size>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>2</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>2</number>
       </property>
       <item>
        <widget class="QToolButton" name="btStudentInfo">
         <property name="minimumSize">
          <size>
           <width>80</width>
           <height>0</height>
          </size>
         </property>
         <property name="text">
          <string>学员信息</string>
         </property>
         <property name="icon">
          <iconset resource="res.qrc">
           <normaloff>:/ico/ICON-8.png</normaloff>:/ico/ICON-8.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>60</width>
           <height>60</height>
          </size>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
         <property name="toolButtonStyle">
          <enum>Qt::ToolButtonStyle::ToolButtonTextUnderIcon</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="btnSchedule">
         <property name="minimumSize">
          <size>
           <width>80</width>
           <height>0</height>
          </size>
         </property>
         <property name="text">
          <string>课程安排</string>
         </property>
         <property name="icon">
          <iconset resource="res.qrc">
           <normaloff>:/ico/WATCH01.ICO</normaloff>:/ico/WATCH01.ICO</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>60</width>
           <height>60</height>
          </size>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
         <property name="toolButtonStyle">
          <enum>Qt::ToolButtonStyle::ToolButtonTextUnderIcon</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="btnFinance">
         <property name="minimumSize">
          <size>
           <width>80</width>
           <height>0</height>
          </size>
         </property>
         <property name="text">
          <string>财务</string>
         </property>
         <property name="icon">
          <iconset resource="res.qrc">
           <normaloff>:/ico/moneybox-32.ico</normaloff>:/ico/moneybox-32.ico</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>60</width>
           <height>60</height>
          </size>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
         <property name="toolButtonStyle">
          <enum>Qt::ToolButtonStyle::ToolButtonTextUnderIcon</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="btnHonor">
         <property name="minimumSize">
          <size>
           <width>80</width>
           <height>0</height>
          </size>
         </property>
         <property name="text">
          <string>荣誉墙</string>
         </property>
         <property name="icon">
          <iconset resource="res.qrc">
           <normaloff>:/ico/SUN.ICO</normaloff>:/ico/SUN.ICO</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>60</width>
           <height>60</height>
          </size>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
         <property name="toolButtonStyle">
          <enum>Qt::ToolButtonStyle::ToolButtonTextUnderIcon</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="btnSystemSetting">
         <property name="minimumSize">
          <size>
           <width>80</width>
           <height>0</height>
          </size>
         </property>
         <property name="text">
          <string>系统设置</string>
         </property>
         <property name="icon">
          <iconset resource="res.qrc">
           <normaloff>:/ico/ICON-2.png</normaloff>:/ico/ICON-2.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>60</width>
           <height>60</height>
          </size>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
         <property name="toolButtonStyle">
          <enum>Qt::ToolButtonStyle::ToolButtonTextUnderIcon</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="weather_1">
         <property name="minimumSize">
          <size>
           <width>80</width>
           <height>0</height>
          </size>
         </property>
         <property name="text">
          <string>天气预报</string>
         </property>
         <property name="icon">
          <iconset resource="res.qrc">
           <normaloff>:/res/type/BaoXue.png</normaloff>:/res/type/BaoXue.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>60</width>
           <height>60</height>
          </size>
         </property>
         <property name="checkable">
          <bool>false</bool>
         </property>
         <property name="toolButtonStyle">
          <enum>Qt::ToolButtonStyle::ToolButtonTextUnderIcon</enum>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Orientation::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>49</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QStackedWidget" name="stackedWidget">
      <property name="currentIndex">
       <number>1</number>
      </property>
      <widget class="StudentInfoWidget" name="pageStudentInfo"/>
      <widget class="SystemSettingsWidget" name="pageSystemSetting"/>
      <widget class="FinancialWidget" name="pageFinance"/>
      <widget class="HonorWallWidget" name="pageHonor"/>
      <widget class="ScheduleWidget" name="pageSchedule"/>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>StudentInfoWidget</class>
   <extends>QWidget</extends>
   <header location="global">studentinfowidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ScheduleWidget</class>
   <extends>QWidget</extends>
   <header location="global">schedulewidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>FinancialWidget</class>
   <extends>QWidget</extends>
   <header location="global">financialwidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>HonorWallWidget</class>
   <extends>QWidget</extends>
   <header location="global">honorwallwidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>SystemSettingsWidget</class>
   <extends>QWidget</extends>
   <header location="global">systemsettingswidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections/>
</ui>
